ambari-server/src/main/resources/common-services/SPARK2/2.0.0/package/scripts/alerts/alert_spark2_thrift_port.py

```diff
@@ -131,9 +148,16 @@ def execute(configurations={}, parameters={}, host_name=None):
             host_name = socket.getfqdn()
 
         if security_enabled:
-            beeline_url = ["'jdbc:hive2://{host_name}:{port}/default;principal={hive_principal}'","transportMode={transport_mode}"]
+            if spark_ssl_enabled:
+                beeline_url = ['"jdbc:hive2://{host_name}:{port}/default;principal={hive_principal};transportMode={transport_mode;ssl=true;sslTrustStore={spark_truststore_path};trustStorePassword={spark_truststore_pass!p};httpPath=cliservice"']
```

Comment:
This seems incorrect 
`transportMode={transport_mode;ssl`

Reply:
thanks @bikassaha for pointing out that miss . I have corrected that and pushed the changes,